shotgun_api3/shotgun.py

```diff
@@ -675,6 +688,32 @@ def _construct_read_parameters(self,
         params["paging"] = { "entities_per_page": self.config.records_per_page,
                              "current_page": 1 }
 
+        if additional_server_presets is not None:
+            if not isinstance(additional_server_presets, list):
```

Comment:
Nit: May want to allow tuples too, e.g.: `if not isinstance(additional_server_presets, (list, tuple)):`